addon/components/markers-styles/image/icon-editor.js

```diff
@@ -177,6 +177,56 @@ export default Ember.Component.extend({
   */
   iconAnchor: null,
 
+  /**
+    Flag: indicates whether to show editor shadow icon image or not.
+
+    @property _disabled
+    @type Boolean
+    @private
+    @default false
+  */
+  _disabled: Ember.computed('iconUrl', function() {
```

Comment:
This property is attached to flexberry-checkbox `value`, so it can't be computed, it must be just plain `true/flase` property.
To call `this._clearIconFile()` there must be defined an observer method `_disabledDidChange: Ember.observer('_disabled', function() { this._clearIconFile(); })`.
Check if iconUrl is defined can be performed only once in component's initialization time: `init() { this._super(...arguments);  if (this.get('allowDisabling') && Ember.isNone(this.get('iconUrl'))) { this.set('_disabled', true) }}`